index.html

```diff
@@ -204,7 +304,73 @@
 						bindKey: { win: 'Ctrl-Enter', mac: 'Command-Enter' },
 						exec: vm.run_code,
 					})
+
+					// Control-S -> Save file
+					vm.editor.commands.addCommand({
+						name: 'save_code',
+						bindKey: { win: 'Ctrl-S', mac: 'Ctrl-S' },
```

Comment:
I'm not a Mac person, but shouldn't this be `Command-S`?

Reply:
updated with fix